### PR TITLE
Update README with contact info, feature addition, and more specific …

### PR DESCRIPTION
**Pull request by stillsheryl**
#### What does this PR do?
Update the README to add specific setup instructions, add a description of our added feature, and add contact information. Add link to credit original repo.

**Note:** We will want to add a picture of our added functionality!

### CODE REVIEW

**Review by s-steel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,14 +3,15 @@
 # Crate 👕👖📦
 
 #### Get monthly subscription of trendy clothes and accessories.
+- This application was built using the framework from a [base application](https://github.com/atulmy/crate) created by [Atul Yadav](https://github.com/atulmy) and other contributors.
```
Comment (stillsheryl) [inline README.md]: We may want to add a link to the project requirements?

**Review by stillsheryl — COMMENTED** [review]
We will want to add a picture of the functionality we added once it's ready!

**Review by stillsheryl — COMMENTED** [review]
We may also want to add a link to our Miro board to show our wireframes and process

File: README.md [hunk 2]
```diff
@@ -21,18 +22,11 @@
...
-- Developers with basic knowledge on React exploring advance React projects
-- Developers learning React Native
-- Exploring GraphQL
-- Scalable project structure and code
-- Starter application for Mobile and Web along with SSR
-- Multi-package scripts
-- Sample project with combination of all above
+- Multi-package setup and dev scripts for an automated dev experience
 
+## Features Added by our Team
```
Comment (dcoleman21) [inline README.md:27]: Love that you highlight the features that were added by our team.

File: README.md [hunk 3]
```diff
@@ -97,18 +91,30 @@ Click on image to view fullscreen and zoom
...
 - Configurations
   - Modify `/api/src/config/database.json` for database credentials
+    1. Check your available postgres users by running \du in the postgres CLI. Choose one of the names (“postgres” should be fine) to put as your username in the database.json.
+    2. If you followed the default installation instructions when install postgres, then you do not need a password. If you have a password created, add that to the database.json file.
+    3. Change the "dialect": "mysql" to "dialect": "postgres"
   - Modify `/api/.env` for PORT (optional)
   - Modify `/web/.env` for PORT / API URL (optional)
   - Modify `/mobile/src/setup/config.json` for API URL (tip: use `ifconfig` to get your local IP address)
 - Setup
+  - Run these commands to install PostgreSQL (if needed), and Sequelize-CLI tool:
```
Comment (dcoleman21) [inline README.md:107]: Great addition of the things that we encountered while setting up.

File: README.md [hunk 4]
```diff
@@ -132,48 +138,24 @@ Click on image to view fullscreen and zoom
...
-- Container Components - [Medium Post](https://medium.com/@learnreact/container-components-c0e67432e005)
-- Zero to GraphQL in 30 Minutes - [YouTube](https://www.youtube.com/watch?v=UBGzsb2UkeY&list=PLkuiMQfg5DujhOSZ1A8kDl0hKV_ICTjp-)
-- Building a GraphQL Server [YouTube Playlist](https://www.youtube.com/playlist?list=PLillGF-RfqbYZty73_PHBqKRDnv7ikh68)
-- Universal JavaScript Web Applications with React by [Luciano Mammino](https://github.com/lmammino) - [YouTube](https://www.youtube.com/watch?v=0VEwRFP8WtI)
-- Building Youtube UI in React Native in 30 Mins - [YouTube](https://www.youtube.com/watch?v=LdKtugH-sb8)
-- Building Stellar Mobile UX With React Native - [YouTube](https://www.youtube.com/watch?v=ssXB9RMTpTs)
-- Free MySQL hosting https://remotemysql.com
+## Database Schema
 
+![crate_schema](https://user-images.githubusercontent.com/7945439/107857326-1dd25d80-6deb-11eb-814f-785fb4c24c24.png)
```
Comment (dcoleman21) [inline README.md:143]: Love that we have our schema image.

File: README.md [hunk 5]
```diff
@@ -132,48 +138,24 @@ Click on image to view fullscreen and zoom
...
-## Donate
-If you liked this project, you can donate to support it ❤️
-
-[![Donate via PayPal](https://raw.githubusercontent.com/atulmy/atulmy.github.io/master/images/mix/paypal-me-smaller.png)](http://paypal.me/atulmy) [![Become a Patreon](https://raw.githubusercontent.com/atulmy/atulmy.github.io/master/images/mix/patreon.png?v=1)](https://www.patreon.com/atulmy)
-
-
-## Hire me
-Looking for a developer to build your next idea or need a developer to work remotely? Get in touch: [atul.12788@gmail.com](mailto:atul.12788@gmail.com)
-
+Dani Coleman - [Github](https://github.com/dcoleman21)
```
Comment (dcoleman21) [inline README.md:153]: You all are great!

**Review by dcoleman21 — APPROVED** [review]
This looks really great!

**Pull request merged**